unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/IdleChecker.cs

```diff
@@ -0,0 +1,96 @@
+using UnityEngine;
+
+namespace DCL
+{
+    public interface IIdleChecker
+    {
+        void Initialize();
+        void Dispose();
+        void SetMaxTime(int time);
+        int GetMaxTime();
+        bool isIdle();
+        void Update();
+        
+        delegate void ChangeStatus(bool isIdle);
+        void Subscribe(ChangeStatus callback);
+        void Unsubscribe(ChangeStatus callback);
+    }
+    
+    public class IdleChecker : IIdleChecker
+    {
+        Vector3 _lastMouseCoordinate = Vector3.zero;
+        private int _maxTime = 60;
+        private float _lastActivityTime = 0.0f;
+        private bool _isIdle = false;
```

Comment:
Let's remove the "_" (underscore) in these variables and type their names in lower camel case, to be consistent with our guidelines